internal-packages/react-graphql-test-app/dataeden.config.mjs

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from '@data-eden/config';
```

Comment:
I think this should just be part of `@data-eden/codegen` rather than being a separate package since it's the config specifically for codegen. Seems weird to have to install another package just to configure codegen

Reply:
it is a config for anything data-eden, that's why the nested field is codegen.